package.json

```diff
@@ -31,7 +31,8 @@
   "scripts": {
     "lint": "eslint bin/* lib test",
     "watch": "chokidar \"lib/**/*.js\" -c \"npm run test\"",
-    "test": "npm run lint && mocha && mocha test/individual && if [ ! \"$TRAVIS\" ]; then snyk test; fi"
+    "test": "npm run lint && mocha && mocha test/individual && if [ ! \"$TRAVIS\" ]; then snyk test; fi",
+    "test:dev": "mocha && mocha test/individual"
```

Comment:
Please don't add scripts you used while testing to your PR. It's an unrelated change.